index.html

```diff
@@ -483,14 +484,15 @@ <h4>Script Type</h4>
               <li>
                 <dfn>Original Language Transcript</dfn>:
                 <p>When the <code>daptm:scriptType</code> value is <code>originalTranscript</code>,
-                the document is a literal transcription of the dialogue and on-screen text in their original spoken/written language(s).</p>
-                <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Original</a>
+                the document is a literal transcription of the dialogue and/or on-screen text in their inherent spoken/written language(s),
+                or of non-dialogue sounds and non-linguistic visual content.</p>
+                <p><a>Script Events</a> in this type of <a>transcript</a> SHOULD contain
+                  <a>Original</a> language <a>Text</a> objects
                   and SHOULD NOT contain <a>Text</a> objects whose
-                  <a>Text Language Source</a> is set to <a>Translation</a>.</p>
+                  <a>Text Language Source</a> indicates that they are <a>Translations</a>.</p>
```

Comment:
That's not a change in this PR. If I recall correctly it is because the Script Events may contain no Text objects at all, in the initial stages of authoring AD for example.

Reply:
Thanks, you're right this should be handled with #52